association.go

```diff
@@ -2943,3 +3031,298 @@ func (a *Association) completeHandshake(handshakeErr error) bool {
 
 	return false
 }
+
+func (a *Association) startRackTimer(dur time.Duration) {
+	if !a.rackEnabled || dur <= 0 {
+		return
+	}
+
+	a.rackTimerMu.Lock()
+	defer a.rackTimerMu.Unlock()
+
+	gen := atomic.AddUint32(&a.rackTimerGen, 1)
+
+	if a.rackTimer != nil {
+		a.rackTimer.Stop()
+	}
+
+	a.rackTimer = time.AfterFunc(dur, func() {
```

Comment:
I'm little curious how frequently the timers are created and how much it affects the performance